src/Estimators/EstimatorManagerInput.cpp

```diff
@@ -22,11 +22,28 @@ EstimatorManagerInput::EstimatorManagerInput(xmlNodePtr cur) { readXML(cur); }
 
 EstimatorManagerInput::EstimatorManagerInput(std::initializer_list<EstimatorManagerInput> emil)
 {
-  std::for_each(emil.begin(), emil.end(), [this](const EstimatorManagerInput& emi) {
-    std::copy(emi.estimator_inputs_.begin(), emi.estimator_inputs_.end(), std::back_inserter(estimator_inputs_));
+  // \todo the following code fusing two vectors can be written in more consice way with std::back_inserter. See history.
+  // Right now needs to use a clumsy way to make intel classic compiler 19.1.x happy when using gcc 9 on some OS distros.
```

Comment:
@PDoakORNL Added TODO note here.
Q: does std::back_inserter potentially over allocate memory?
I'm not saying memory is a concern here. Just to understand how back_inserter works.

Reply:
If there are concerns we can put an #ifdef for this particularly compiler, just as we have in the past. There are still alot of Intel 19.x compilers in use, so if we can make things easier for users easily we should do so.

Reply:
Considering the workaround is localized the think the \todo is less trouble.

Reply:
And back_inserter works like the containers push_back.